qmf/src/plugins/messageservices/imap/imapauthenticator.cpp

```diff
@@ -52,11 +52,11 @@
 
 namespace {
 
-QMap<QMailAccountId, QList<QByteArray> > gResponses;
-
 #ifdef USE_ACCOUNTS_QT
 QString authPassword;
 QMail::SaslMechanism responseAuthType;
+QList<QByteArray> gResponses;
+
```

Comment:
This was not really used anymore, so just reusing here to be similar to smtp authenticator